app/templates/navbar.html

```diff
@@ -24,11 +24,16 @@
         <a class="dropdown-item nav-link bg-dark" href="/new_zseries_recipe">New</a>
       </div>
     </li>
-    <li class="nav-item"><a class="nav-link" href="/brew_history">Brew History</a></li>
-    <li class="nav-item"><a class="nav-link" href="/ferm_history">Ferm History</a></li>
+    <li class="nav-item dropdown">
+      <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Sessions</a>
+      <div class="dropdown-menu dropdown-menu-right navbar-dark bg-dark" aria-labelledby="navbarDropdown">
+        <a class="dropdown-item nav-link bg-dark" href="/brew_history">Brew History</a>
+        <a class="dropdown-item nav-link bg-dark" href="/ferm_history">Ferm History</a>
+      </div>
+    </li>

```

Comment:
choose to clean up and instead hiding these under a single dropdown menu -> 
![image](https://user-images.githubusercontent.com/2158627/88352605-506e1900-cd28-11ea-9224-4aa029fc55ff.png)